File: src/main/java/com/linebot/controller/keep/KeepAliveController.java
package com.linebot.controller.keep;

import com.linebot.model.response.RestApiResponse;
import lombok.AllArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@Slf4j
@RestController
@AllArgsConstructor
@RequestMapping(path = "/self")
public class KeepAliveController {
    @GetMapping("/keep")
    public RestApiResponse index() {
        log.info("keep alive.");
        return RestApiResponse.builder()
                .status("success")
                .build();
    }
}

File: src/main/kotlin/com/linebot/api/keep/KeepAliveController.kt
package com.linebot.api.keep

import com.linebot.model.response.RestApiResponse
import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping(path = ["/self"])
class KeepAliveController {
    val log: Logger = LoggerFactory.getLogger(KeepAliveController::class.java)

    @GetMapping("/keep")
    fun index(): RestApiResponse {
        log.info("keep alive.")
        return RestApiResponse("success")
    }
}
